pkg-r/R/querychat_module.R

```diff
@@ -0,0 +1,104 @@
+# Main module UI function
+mod_ui <- function(id) {
```

Comment:
This needs to take and pass `...` as documented in `$ui()`

Reply:
66a2953076d0e037843b2afd6b6b4a69ef0be2de